tests/tnimlsp.nim

```diff
@@ -26,10 +26,10 @@ suite "Nim LSP basic operation":
         workspaceFolders = none(seq[WorkspaceFolder])
       ).JsonNode)
     ).JsonNode
-    i.sendJson ir
+    discard i.sendJson ir
```

Comment:
You really shouldn't discard an async, should probably be changed to a `waitFor`.